project/src/test/java/com/google/sps/SearchStoreTest.java

```diff
@@ -353,4 +424,116 @@ public void twoEvents_tieInRelevanceInDescriptionOfBothEvents_twoResultsReturned
     Assert.assertEquals(
         DESCRIPTION_WITH_GAMES_IN_HIGH_RELEVANCE, actualResults.get(1).getDescription());
   }
+
+  /**
+   * When an event with description with the keyword food vendors is added to the index,
+   * a search for the word food should also return the event in result. Food is a 
+   * word within the keyword food vendors.
+   */
+  @Test
+  public void
+      addEventWithKeywordInDescription_searchForWordWithinKeyword_oneResultReturned()
+          throws IOException {
```

Comment:
You can throws Exception here. Don't need to be so specific in the test.

Reply:
Done.